src/app/pages/tech/tech.component.ts

```diff
@@ -9,14 +9,52 @@ import { TechService, ITech } from '../tech/tech.service';
 export class TechComponent implements OnInit {
 
   title = "Tech"
+  techFilter: any[] = [];
+  nameFilter: string = "";
   tech: ITech[] = [];
+  filteredTech: ITech[] = [];
+  class: classCheckbox = [];
 
   constructor(
     private techService: TechService
   ) { }
 
+
```

Comment:
Can we make this into a separate service at app/shared/filter.service.ts similarly to tiles.service.ts?